x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/plugin/MatchOperatorIT.java

```diff
@@ -128,6 +129,48 @@ public void testNotWhereMatch() {
         }
     }
 
+    public void testWhereMatchWithScoring() {
```

Comment:
It will be great to add some more `_score` tests into `CsvTests`, there were some `QSTR` related tests added recently in `qstr-function.csv-spec`, it could be a good place to add more `_score` related tests.

Reply:
thanks Fang, I'll do it ;)